swagger/v1/location.json

```diff
@@ -57,5 +58,65 @@
         }
       }
     }
+  },
+  "Location2": {
```

Comment:
Why are there two Location types now?

Reply:
there are two locations (`Location1` and `Location2`, but only one type, `location`) - this is to make the examples a bit more realistic, so they can have a from_location and to_location which are different.

Reply:
Right, got it - thanks for the explanation 👍